data/linux-bridge/002-linux-bridge.yaml

```diff
@@ -39,23 +39,23 @@ spec:
             - /bin/bash
             - -ce
             - |
-              echo 'Installing bridge and tuning CNIs'
               cni_mount_dir=/opt/cni/bin
-              sourcebinpath=/usr/src/github.com/containernetworking/plugins/bin
-              cp --remove-destination ${sourcebinpath}/bridge ${cni_mount_dir}/cnv-bridge
-              cp --remove-destination ${sourcebinpath}/tuning ${cni_mount_dir}/cnv-tuning
+              if find ${cni_mount_dir}/bridge &>/dev/null && ! test -L ${cni_mount_dir}/bridge; then
```

Comment:
Let's not make assumptions about the tuning also accompanying bridge. Please split this into two if-elses, one for bridge, one for tuning.

Also please add echos explaining what happens and why, to make it easier for cluster admins to understand what's going on (and what are our intentions) while looking in the logs.

Reply:
Done